### ISSUE TITLE
Pagination

### ISSUE THREAD

**Issue by andorp** [liqd/aula#271]
Many pages enumerate data. There is no pagination defined in the click dummy. A list of pages should be created where the pagination applies.

**Comment by rittermo** [comment]
You’re right @andorp. I’ll put it in the list of additions that Nadine should make to the clickdummy. Since she will not be here until mid-April, I’ll probably ask our other designer, Nico, to take a shot at it. I’ll update you, once it is in the clickdummy.

**Comment by fisx** [comment]
we don't do that until we have a practial reason.

wait, why don't we do this: deliver the un-paginated page, and run js code that hides all but the first page, and unravels the rest as the user scrolls into it.
